src/test/java/seedu/address/testutil/BuyerBuilder.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.testutil;
+
+import seedu.address.model.field.Person;
+import seedu.address.model.field.Price;
+import seedu.address.model.property.Buyer;
+
+public class BuyerBuilder extends PersonBuilder {
+
+    public static final String DEFAULT_PRICE = "1230000";
+
+    private Price max;
```

Comment:
Could this be changed to `maxPrice` as well to be consistent?

Reply:
Sure, fixed